posts/2023-02-27-introducing-new-conda.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Introducing the new hh5 conda environment
+categories: python
+---
+# Introducing the new hh5 conda environment
+
+## Background
+The hh5 python/conda environment has grown vastly in scope since it was first installed many years ago. What started as a place to put important python packages missing from NCI's installations has since become an agile, community-driven, monolithic installation serving almost 600 researchers. Currently, there are 261 packages listed in the file used to construct and update the environment. Once all of the dependencies of these packages are factored in, there are 1,015 total python packages installed in the current `analysis3-unstable` environment. As a result of the work put in by my predecessors at CLEX CMS, an update to the `analysis3-unstable` environment, from a researcher requesting a package through cws_help@nci.org.au to integration and testing of that package, can be completed in under 1 hour.

```

Comment:
```suggestion
The hh5 python/conda environment has grown vastly in scope since it was first installed many years ago. What started as a place to put important python packages missing from NCI's installations has since become an agile, community-driven, monolithic installation serving almost 600 researchers. Currently, there are 261 packages listed in the file used to construct and update the environment. Once all the dependencies of these packages are factored in, there are 1,015 total python packages installed in the current `analysis3-unstable` environment. As a result of the work put in by my predecessors at CLEX CMS, an update to the `analysis3-unstable` environment, from a researcher requesting a package through cws_help@nci.org.au to integration and testing of that package, can be completed in under 1 hour.

```